hiera/hieradata/dev/wso2/wso2as/5.3.0/manager.yaml

```diff
@@ -0,0 +1,34 @@
+---
+wso2::mgt_hostname : mgt.as.wso2.com
+
+wso2::clustering :
+    enable : true
+    local_member_host : 192.168.100.13
```

Comment:
It might be better to add a comment and describe the purpose of adding a static IP address here.